crates/egui/src/context.rs

```diff
@@ -436,16 +473,23 @@ impl Context {
 
         self.check_for_id_clash(id, rect, "widget");
 
+        #[cfg(feature = "accesskit")]
+        {
+            if sense.focusable {
+                // Make sure anything that can receive focus has an AccessKit node.
+                // TODO(mwcampbell): For nodes that are filled from widget info,
+                // some information is written to the node twice.
+                let mut node = self.accesskit_node(id, None);
+                response.fill_accesskit_node_common(&mut node);
+            }
+        }
+
         let clicked_elsewhere = response.clicked_elsewhere();
         let ctx_impl = &mut *self.write();
         let memory = &mut ctx_impl.memory;
         let input = &mut ctx_impl.input;
 
-        // We only want to focus labels if the screen reader is on.
```

Comment:
I think we should keep this behavior. Tab to select the next button is what I ususally want from a GUI. Selecting labels is only interesting if the screen reader is on

Reply:
Please see the corresponding change I made in the label widget. Short version: the label widget now adjusts its sense setting based on whether the built-in screen reader is on.

Reply:
Oh I see. Why did you opt to handle that behavior in `Label` rather than here?

Reply:
Since the label widget is the one special case, I thought it made sense to encapsulate the special logic in that widget. And this way, the AccessKit integration code can just look at `sense.focus` without having to know about the label special case. Does that make sense?